modules/rostests/apitests/win32nt/ntuser/NtUserConvertMemHandle.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for NtUserConvertMemHandle
+ * COPYRIGHT:   Copyright 2025 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include "../win32nt.h"
+
+
+START_TEST(NtUserConvertMemHandle)
+{
+    HANDLE hMem;
+    CHAR data[] = {1, 2, 3, 4};
+ 
+    hMem = NtUserConvertMemHandle((PVOID)(UINT_PTR)0xDEADBEEF, 0xFFFF);
+    ok_hdl(hMem, NULL);
+
+    hMem = NtUserConvertMemHandle(data, sizeof(data));
+    ok(hMem != NULL, "hMem is NULL\n");
+    // FIXME: Clean up hMem.
```

Comment:
You need to invoke SetClipboardData (or better, NtUserSetClipboardData if you want to manipulate the NtUser* functions directly) so as to let the system manage the hMem handle.
(See examples there: https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/user32/windows/clipboard.c;hb=67e5343dafc24626038f8b32cec9d5c82443288f#l297 )